services/worker/src/worker/config.py

```diff
@@ -24,15 +24,15 @@ def get_empty_str_list() -> List[str]:
 
 
 @dataclass(frozen=True)
-class WorkerLoopConfig:
+class LoopConfig:
     max_disk_usage_pct: int = WORKER_LOOP_MAX_DISK_USAGE_PCT
     max_load_pct: int = WORKER_LOOP_MAX_LOAD_PCT
     max_memory_pct: int = WORKER_LOOP_MAX_MEMORY_PCT
     sleep_seconds: int = WORKER_LOOP_SLEEP_SECONDS
     storage_paths: List[str] = field(default_factory=get_empty_str_list)
 
     @classmethod
-    def from_env(cls) -> "WorkerLoopConfig":
+    def from_env(cls) -> "LoopConfig":
         env = Env(expand_vars=True)
         with env.prefixed("WORKER_LOOP_"):
```

Comment:
```suggestion
        with env.prefixed("LOOP_"):
```

Reply:
But I am not sure, we might need to change all other WORKER_ configs as well :/. Maybe keep it as it is

Reply:
You're right, I have to change in the rest of the code

Reply:
well, I will let as WORKER_LOOP better in the code as it's more descriptive. It's not coherent, maybe we should let worker_loop.py in the code...